config/manifests/kustomization.yaml

```diff
@@ -3,7 +3,7 @@
 resources:
 - bases/trustee-operator.clusterserviceversion.yaml
 - ../default
-- ../samples
+# - ../samples
```

Comment:
Can you check what happens if you run make bundle again?
Does the samples get regenerated and should there be an instruction in the Dockerfile to delete it when creating the bundle image - https://github.com/confidential-containers/trustee-operator/blob/main/bundle.Dockerfile ?

Reply:
Make bundle doesn't regenerate the samples as expected.